src/SupabaseStorageClient.ts

```diff
@@ -15,3 +15,8 @@ export class SupabaseStorageClient extends StorageBucketApi {
     return new StorageFileApi(this.url, this.headers, id, this.fetch)
   }
 }
+
+export {
+  SupabaseStorageClient as SupabaseStorageClient,
+  SupabaseStorageClient as StorageClient,
+}
```

Comment:
Nit: retain the new line at the end of file.

Reply:
Agreed. I also like to end all my files with newlines. I must've missed it 😅 

[Fixed](https://github.com/supabase/storage-js/pull/48/commits/0a13fd3782763dba232668d90271816e9b73126e)